lambdas.yml

```diff
@@ -4,3 +4,21 @@ HelloWorld:
   memory: 256
   source: 'node_modules/@cumulus/hello-world/dist/'
   useMessageAdapter: true
+
+DiscoverPdrs:
+  handler: index.handler
+  timeout: 300
+  source: 'node_modules/@cumulus/discover-pdrs/dist/'
+  useMessageAdapter: true
+  logToElasticSearch: true
+  envs:
+    internal: '{{buckets.internal}}'
+
+QueuePdrs:
+  handler: index.handler
+  timeout: 300
+  source: 'node_modules/@cumulus/queue-pdrs/dist/'
+  useMessageAdapter: true
+  logToElasticSearch: true
+  envs:
+    internal: '{{buckets.internal}}'
```

Comment:
I don't think this env variable is used in this task

Reply:
👍